.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
+/crates/*/target
```

Comment:
I did yeah - I just tried to reproduce from a clean checkout though and didn't get it this time. The only difference is that I've updated to the latest stable toolchain since then. I'll remove this.